sklearn/tree/_partitioner.pxd

```diff
@@ -10,7 +10,8 @@ from ._splitter cimport SplitRecord
 
 
 # Mitigate precision differences between 32 bit and 64 bit
-cdef float32_t FEATURE_THRESHOLD = 1e-7
+# Note: Has to be initialized in pyx file, not in the pxd file
```

Comment:
I am not too sure this comment is really needed (we are likely not going to add a similar comment in each Cython module-level variable) but at the same time the behaviour is very suprising (full disclosure: I am definitely not a Cython expert)